src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,17 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: 
+    transform 0.3s ease, 
+    color 0.3s ease;
```

Comment:
Also, you could move `transition-duration` to CSS variables to avoid value duplication.